operator/scripts/olm-testing.sh

```diff
@@ -3,6 +3,10 @@ set -euxo pipefail
 
 UUID=${1:-$(git rev-parse --short HEAD)}
 
+INSTALL_NAMESPACE=${2:-default}
+
+TARGET_NAMESPACES=${3-$INSTALL_NAMESPACE}
```

Comment:
This is actually currently unused, at least not used by the CI. Do we need it?

Reply:
That was mentioned on the other pr - those chagnes are optional.